src/countries/index.test.ts

```diff
@@ -29,6 +29,11 @@ describe('country helpers', () => {
       expect(getCountryByName('Russia')).toMatchSnapshot();
       expect(getCountryByName('Palestine')).toMatchSnapshot();
     });
+
+    it('returns country info by ioc', () => {
```

Comment:
what is ioc?

Reply:
i'm not sure why we need this `ioc`. what's the difference between this and the short name?

Reply:
IOC is the official country code of the International Olympic Committee. You can find this on Wikipedia [Country IOC](https://en.wikipedia.org/wiki/List_of_IOC_country_codes).  You can find this on every country key here `src/countries/list.ts` file.
### Sample Ioc code 
| Country      | IOC |
| ----------- | ----------- |
| United States      | USA       |
| Great Britain   | GBR        |
| Australia   | AUS        |

if a country name contains this IOC short name we can find the config easily.

Reply:
Isn't it same as Short code? do we need this?

Reply:
This list is hard to maintain. To me it seems short code and this is same. Hence, we've two options

1) We can use short code. This is already implemented
2) In mobile project, we can determine Full Country Name and use that.